tools/alpine/Dockerfile

```diff
@@ -56,15 +56,12 @@ RUN go get -u github.com/LK4D4/vndr
 # `test/pkg/containerd/Dockerfile` when changing this.
 # For v1.3.x cherry-pick a upstream commit to allow disabling some tests.
 ENV CONTAINERD_REPO=https://github.com/containerd/containerd.git
-ENV CONTAINERD_COMMIT=v1.3.2
+ENV CONTAINERD_COMMIT=v1.3.3
```

Comment:
Given that we know we need to bump it to v1.3.4 for the fifo fix, why not do it here?

Reply:
because it was released yesterday and I didn't feel like rebasing 20 odd patches and rebuilding everything again (otherwise commits won't line up). better have this in (assuming it works out) and then update containerd seperately.

Reply:
Didn't realize it was quite the headache. No problem then.